frontend/src/components/DomainDetails.tsx

```diff
@@ -20,7 +21,7 @@ import {
 import { Domain } from 'types';
 import { useDomainApi } from 'hooks';
 import { DefinitionList } from './DefinitionList';
-import { differenceInCalendarDays, parseISO } from 'date-fns';
+import { parseISO } from 'date-fns';
```

Comment:
Combine these two import statements (lines 2 and 24)

Reply:
Done